client/dashboard/utils/site-features.ts

```diff
@@ -9,7 +9,7 @@ import type {
 
 // Returns whether the plan supports a specific feature.
 export function hasPlanFeature(
-	site: Site,
+	site: { plan?: { features: { active: string[] } } },
```

Comment:
Just curious why the signature change

Reply:
I needed `hasPlanFeature` to accept either a `Site` or `SiteDetails` object. The `plan.features` part is common between the two so it made sense to me to just accept an object with the data the function requires.